girder/girder_large_image/rest/large_image_resource.py

```diff
@@ -445,6 +447,55 @@ def _deleteCachedImages(self, spec, associatedImages=False, imageKey=None):
                 removed += 1
         return removed
 
+    @access.user(scope=TokenScope.DATA_WRITE)
+    @autoDescribeRoute(
+        Description('Create large images for all items within a folder.')
+        .notes('Does not work for items with multiple files.')
+        .modelParam('id', 'The ID of the folder.', model=Folder, level=AccessType.WRITE, required=True)
+        .param('createJobs','Whether a job should be created for each large image.', required=False, 
+            default=False, dataType='boolean')
+        .param('localJobs', 'Whether a the jobs created should be local.', required=False, 
+            default=False, dataType='boolean')
+        .param('recurse', 'Whether child folders should be recursed', required=False, default=False, 
+            dataType='boolean')
+        .errorResponse('ID was invalid.')
+        .errorResponse('Write access was denied for the folder.', 403)
+    )
+    def createLargeImages(self, folder, params):
+        user=self.getCurrentUser()
+        createJobs = params.get('createJobs')
+        if createJobs:
+            createJobs = 'always' or True
+        self.createImagesRecurseOption(folder=folder, createJobs=createJobs, user=user, 
+                                recurse=params.get('recurse'), localJobs=params.get('localJobs'))
+                    
+    def createImagesRecurseOption(self, folder, createJobs, user, recurse, localJobs):
+        count = {'large images created': 0}
```

Comment:
I don't see where `count` is used. Is this intended to provide summary messages back to the user? We should include things like files skipped, folders recursed, and other info about the behavior.

Reply:
While working I had printed count and just missed this line when cleaning up, but returning a summary message is a great idea!

Reply:
Added in [this commit](https://github.com/girder/large_image/pull/1572/commits/ef19b736766a8f3410552291b78805fc62bc4563)